components/private_channel/rust/ffi/src/lib.rs

```diff
@@ -0,0 +1,172 @@
+#![allow(clippy::missing_safety_doc)]
+extern crate brave_private_channel;
+
+use std::convert::TryInto;
+use std::ffi::CStr;
+use std::os::raw::{c_char, c_int};
+use std::slice;
+
+pub const KEY_SIZE: usize = 32;
+
+trait ErrorResult {
+    fn default() -> Self;
+}
```

Comment:
I'd recommend either to use [`Default`](https://doc.rust-lang.org/std/default/trait.Default.html) from the standard library, or choose a different method name if you'd like to reserve the opportunity to have separate `Default` semantics in the future.

(You can technically implement both `Default::default()` and `ErrorResult::default()` at the same time, but it's annoying and potentially confusing to disambiguate)

Reply:
- [ ] Use `Default::default()` instead of `ErrorResult::default()`